packages/features/ee/organizations/components/AboutOrganizationForm.tsx

```diff
@@ -0,0 +1,117 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import z from "zod";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Avatar, Button, Form, ImageUploader, Alert, Label, TextAreaField } from "@calcom/ui";
+import { ArrowRight } from "@calcom/ui/components/icon";
+
+const querySchema = z.object({
+  id: z.string(),
+});
+
+export const AboutOrganizationForm = () => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const { id: orgId } = querySchema.parse(router.query);
+  const [serverErrorMessage, setServerErrorMessage] = useState<string | null>(null);
+  const [image, setImage] = useState("");
+
+  const aboutOrganizationFormMethods = useForm<{
+    logo: string;
+    bio: string;
+  }>();
+
+  const updateOrganizationMutation = trpc.viewer.organizations.update.useMutation({
+    onSuccess: (data) => {
+      if (data.update) {
+        router.push(`/settings/organizations/${orgId}/onboard-admins`);
+      }
+    },
+    onError: (err) => {
+      setServerErrorMessage(err.message);
+    },
+  });
+
+  return (
+    <>
+      <Form
+        form={aboutOrganizationFormMethods}
+        handleSubmit={(v) => {
+          if (!updateOrganizationMutation.isLoading) {
+            setServerErrorMessage(null);
+            updateOrganizationMutation.mutate({ ...v, orgId });
+          }
+        }}>
+        <div className="mb-5">
```

Comment:
very very minor NIT: we can add space-y-5 to form to acheive the same as all of these `mb-5` divs

Reply:
Done